tutorials/spark/df003_profiles.py

```diff
@@ -0,0 +1,50 @@
+## \file
+## \ingroup tutorial_dataframe
+## \notebook -nodraw
+## This tutorial illustrates how to use TProfiles in combination with the
+## RDataFrame. See the documentation of TProfile and TProfile2D to better
+## understand the analogy of this code with the example one.
+##
+## \macro_code
+##
+## \date February 2017
+## \author Danilo Piparo
+
+import ROOT, PyRDF
+PyRDF.use("spark")
```

Comment:
Should we should in some tutorial how to configure some Spark parameters, or the number of partitions?

Reply:
Yup @etejedor . It'd be better if I include that in one of the tutorials. I'll do it in this tutorial (`df003_profiles.py`) 👍